packages/extension-link/package.json

```diff
@@ -38,7 +38,7 @@
     "prosemirror-state": "^1.4.1"
   },
   "dependencies": {
-    "linkifyjs": "^3.0.5"
+    "linkifyjs": "^4.1.0"
```

Comment:
I guess this major version change won't lead to issues?

Reply:
1. Tiptap doesn't use any removed plugins or custom ones. No issues there
2. Links that start with `mailto:` now have `link` type instead of `email`. We don't have checks based on link type but may be a potential breaking change if someone imported linkify as a transitive dependency. It's a pretty bad pattern so we may ignore it or add some note to the changelog
3. Dropped ie11 and safari10 support. I hope no issues with it